### ISSUE TITLE
2.11.0-beta1 release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3564]
- #3555
- #3420 
- #3562

**Comment by joeparsons** [comment]
## Pick list
```
git cherry-pick 383e443389373c2caeaf0f1690298328f98ea290
git cherry-pick 5d110eb97fef8c5d67db39f0e686b6ae1d43015e
git cherry-pick 7e6eabb51fe42d08971dfd017bd226012a401605
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.11.0-alpha2..HEAD --pretty='format:- %s'`
```
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2. Pre-releases are not supported for production sites but are intended for widespread testing in preparation for the upcoming stable release._

_**Quickstart 2.9.x will no longer receive security support. Sites on Quickstart 2.9.x or earlier should upgrade to a supported release as soon as possible.** See our [release policy](https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#release-overview) for more information._

_For more information about the changes in this release, please see the [project page](https://github.com/orgs/az-digital/projects/106) and the [milestone](https://github.com/az-digital/az_quickstart/milestone/21)._

- Closes #3561: Add NetID field to AZ Person. (#3562)
- Close #3262 Add ability for content admins to set finder overrides. (#3420)
- Update drupal/migrate_plus requirement from 6.0.2 to 6.0.4 (#3555)
- Back to dev.
```

**Comment by joeparsons** [comment]
Released:
https://github.com/az-digital/az_quickstart/releases/tag/2.11.0-beta1
